docs/usage.rst

```diff
@@ -95,8 +93,8 @@ The full list of arguments can be accessed via :bash:`pyreisejl/utility/call.py
 .. code-block:: text
 
    usage: call.py [-h] [-s START_DATE] [-e END_DATE] [-int INTERVAL] [-i INPUT_DIR]
-   [-x EXECUTE_DIR] [-t THREADS] [-d] [-o OUTPUT_DIR] [-m MATLAB_DIR] [-k]
-   [--solver SOLVER] [-j JULIA_ENV] [-l LINEARIZATION_SEGMENTS]
+   [-t THREADS] [-d] [-o [OUTPUT_DIR]] [-k]
```

Comment:
Why do we need brackets around `OUTPUT_DIR`?

Reply:
If optional arguments are surrounded by brackets, then `THREADS` needs brackets too.

Reply:
I think it's based on whether the `nargs` and `default` options are passed to `add_argument` when defining the parser. We don't have those for threads, but it nevertheless defaults to `None`.. I'll see if we can add that so the `--help` shows the brackets in that case.